### PR TITLE
game() initializes 4 empty Arraylist<Card>()

### PR DESCRIPTION
**Pull request by SeanCramsey**

### CODE REVIEW

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -16,6 +16,9 @@
 
     public Game(){
     // initialize a new game such that each column can store cards
+      for ( int i = 0; i < 4; i++){
+        cols[i] = new ArrayList<Card>();
```
Comment (wwang126) [inline src/main/java/models/Game.java]: Arraylist doesn't use the bracket operator.

File: src/main/java/models/Game.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
     public Game(){
     // initialize a new game such that each column can store cards
       for ( int i = 0; i < 4; i++){
-        cols[i] = new ArrayList<Card>();
+        this.cols.get(i).add(new ArrayList<Card>());
```
Comment (wwang126) [inline src/main/java/models/Game.java]: Does this get a pointer or a copy of the object?

File: src/main/java/models/Game.java [hunk 3]
```diff
@@ -17,7 +17,7 @@
     public Game(){
     // initialize a new game such that each column can store cards
       for ( int i = 0; i < 4; i++){
-        this.cols.get(i).add(new ArrayList<Card>());
+        this.cols.add(i, new ArrayList<Card>());
```
Comment (wwang126) [inline src/main/java/models/Game.java]: looks good!

Comment (idot107) [reply src/main/java/models/Game.java]: good job

**Pull request merged**